templates/sway-test-rs/README.md

```diff
@@ -6,6 +6,6 @@ testing within an existing Sway project.
 This template is designed for Rust developers who wish to test integration of
 their Rust application and Sway code.
 
-See the "Testing with Rust" chapter of [the Sway
-Book](https://fuellabs.github.io/sway/v0.25.0/) for a thorough guide on how to
+See the `Testing with Rust` chapter of [the Sway
```

Comment:
I suggest we link to that chapter directly: https://fuellabs.github.io/sway/master/testing/testing-with-rust.html. This is not ideal because it links to the `master` book (we can't have `latest` in the link sadly) but maybe that's okay given that this README _is_ on `master`?

Reply:
Yh I agree, I tried it with `latest` but forgot about `master`. I'll keep the link to `the sway book` so it's always up-to-date and add the chapter specific link as suggested